packages/xrpl/test/setupClient.ts

```diff
@@ -30,6 +31,10 @@ async function setupMockRippledConnection(
     // We must have an error listener attached for reconnect errors
   })
 
+  context.client.on('connected', () => {
+    context.mockRippled?.addResponse('server_info', rippled.server_info.normal)
```

Comment:
Note: this fixture does not include a network ID, but all networks other than mainnet include a network ID.